packages/tools/kadena-cli/src/prompts/network.ts

```diff
@@ -247,6 +251,31 @@ export const networkSelectOnlyPrompt: IPrompt<string> = async (
   return selectedNetwork;
 };
 
+export const networkSelectWithNonePrompt: IPrompt<string> = async (
+  previousQuestions,
+  args,
+  isOptional,
+): Promise<string> => {
+  const existingNetworks = await getEnsureExistingNetworks();
+
+  const choices: ICustomNetworkChoice[] = existingNetworks.map((network) => ({
+    value: network.value,
+    name: network.name,
+  }));
+
+  choices.unshift({
+    value: 'none',
+    name: 'None',
```

Comment:
The `name` could be more descriptive like "Remove default network", and does the option only show up if a default network was already set?